daemon/logdrivers_windows.go

```diff
@@ -0,0 +1,7 @@
+package daemon
```

Comment:
Don't understand this.
Don't logdrivers_linux and logdrivers_windows need build tags?